### DISCUSSION TITLE
Display "Pages that use the shortcode" is faulty

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#530] [_Discarded pitches]
Current state

At the moment it seems that the form & list shortcodes are permanently displayed for all languages. For example, if there is a form in DE and EN on the home page, "Home" is displayed 2 times.

![anzeige-shortcode1](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/88546396/7a3675e0-f6d2-4cde-b4b5-d16283d6dbe2)

### Desired state

Only the pages for the currently selected / relevant language should be displayed.

**Comment by fredericalpers** [comment]
@yeneastgate please investigate during ramp up. if you guys can find a solution fast go ahead and create an issue.

**Reply by yeneastgate** [reply]
Yes, I will investigate it and let you know soon.

**Reply by yeneastgate** [reply]
@fredericalpers 
After researching this task, I don't think we can find a solution quickly.
Because currently, WordPress core does not have the function to check language for page by "page_id".
We can only check language when we add new a WordPress Multilingual Plugin(wpml, Polylang..)
I'm sorry, we will continue it in the next cycle.

**Reply by fredericalpers** [reply]
@yeneastgate Okay! Thank you for investigating. :) I will lock the disscussion until then.
